pkg/drivers/kic/oci/oci.go

```diff
@@ -702,7 +702,9 @@ func DaemonHost(driver string) string {
 		if dh := os.Getenv(constants.DockerHostEnv); dh != "" {
 			if u, err := url.Parse(dh); err == nil {
 				if u.Host != "" {
-					return u.Hostname()
+					if hostname := u.Hostname(); hostname != "" {
```

Comment:
The above `if u.Host != "" {` seems redundant since an empty `u.Host` returns an empty `u.Hostname()` as well. Can simplify it to just make sure the return value is not empty.

```
if u, err := url.Parse(dh); err == nil {
        if u.Hostname() != "" {
                return u.Hostname()
        }
}
```

Reply:
Done - should I also update the podman code above to just use u.Hostname?

Reply:
Yeah, it would likely suffer from the same issue